backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/ProprietaryService.java

```diff
@@ -80,4 +80,25 @@ public Proprietary updateProprietaryFromNorm(UpdateProprietaryFromNormUseCase.Qu
 
     return proprietary;
   }
+
+  @Override
+  public Proprietary updateProprietarySingleElementFromNorm(
+      UpdateProprietarySingleElementFromNormUseCase.Query query) throws NormNotFoundException {
```

Comment:
Also remove the `throws NormNotFoundException` here. Not longer needed, as after the PR merge from Sebastian this exception is now an unchecked exception